content/modules/rest/pages/entities-api/update-entities.adoc

```diff
@@ -0,0 +1,404 @@
+      "createdDate": "2021-03-05T10:35:13.427",
+      "createdBy": "admin",
+      "lastModifiedDate": "2021-03-05T10:35:13.427",
+      "version": 1
+    },
+    {
+      "_entityName": "rstex11_OrderLine",
+      "_instanceName": "rest.sample.entity.OrderLine-55b925e5-9f3a-a725-9eb3-1240f9c1fe95 [detached]",
+      "id": "55b925e5-9f3a-a725-9eb3-1240f9c1fe95",  // <2>
+      "product": {
+        "_entityName": "rstex11_Product",
+        "_instanceName": "Cotek Battery Charger",
+        "id": "1ed85c7a-89f1-c339-a738-16307ed6003a",
+        "name": "Cotek Battery Charger"
+      },
+      "quantity": 1.0,
+      "createdDate": "2021-03-05T10:35:13.427",
+      "createdBy": "admin",
+      "lastModifiedDate": "2021-03-05T10:35:13.427",
+      "version": 1
+    }
+  ],
+  "version": 1,
+  "customer": {
+    "_entityName": "rstex11_Customer",
+    "_instanceName": "Randall Bishop",
+    "id": "f88597ff-009d-1cf2-4a90-a4fb5b08d835",
+    "createdDate": "2021-03-01T08:33:25.326",
+    "createdBy": "admin",
+    "lastModifiedDate": "2021-03-01T08:33:25.326",
+    "name": "Randall Bishop",
+    "version": 1
+  }
+}
+----
+<1> The first order line references the `Solar-One HUP Flooded Battery 48V` product.
+<2> The second order line references the `Cotek Battery Charger` product.
+
+In this example you would like to change the order lines in the following ways:
+
+* The `quantity` of the order line with the product `Solar-One HUP Flooded Battery 48V` should be increased to `3.0`.
+* The order line with the product `Cotek Battery Charger` should be removed.
+* A new order line with the product `Outback Power Remote Power System` should be added.
+
+[source, http request]
+.Update Composition Request
+----
+PUT http://localhost:8080/rest
+            /entities
+            /rstex11_Order
+            /288a5d75-f06f-d150-9b70-efee1272b96c
+            ?responseFetchPlan=product-with-tags
+
+{
+  "customer": {
+    "id": "f88597ff-009d-1cf2-4a90-a4fb5b08d835"
+  },
+  "date": "2021-03-01",
+  "amount": 249.99,
+  "lines": [
+    {
+      "id": "a1cd778b-fe49-4c74-05a0-6fb207dc11bd", // <1>
+      "product": {
+        "id": "1860904a-5444-9c3e-9dc1-1d7a26d9ac19",
+        "name": "Solar-One HUP Flooded Battery 48V"
+      },
+      "quantity": 3.0 // <2>
+    },
+    { // <3>
+      "product": {
+        "id": "f6884077-19c4-546f-33d4-a788399337f7",
+        "name": "Outback Power Remote Power System"
+      },
+      "quantity": 1.0
+    }
+  ]
+}
+----
+<1> The ID of the existing order line is added to update the existing order line
+<2> The `quantity` value is set to `3.0` for the `Solar-One HUP Flooded Battery 48V` product
+<3> A new order line is added for the product `Outback Power Remote Power System`
+
+WARNING: When updating a child entity, like the order line in the example above, the ID of the existing order line needs to be added, so that Jmix recognises it as an update. Otherwise, it would treat the child entity as a new entity.
```

Comment:
Let's limit or diversity of using admonition styles to NOTE, TIP, CAUTION and use WARNING only when there is really great potential harm. Here NOTE or TIP seems to be enough.